sass/base/_base.scss

```diff
@@ -14,7 +14,7 @@ html, input {
   // 16px = 100%
   // 18px = 112.5%
   // See http://pxtoem.com/ for conversion chart
-  font-size: (16 / $cd-font-size-base * 100) + %;
+  font-size: ($cd-font-size-base / 16 * 100) + %;
```

Comment:
A few common examples:

* 18 / 16 x 100 = 112.5%
* 10 / 16 x 100 = 62.5%

Reply:
Thanks. This was wrecking my head :)